src/main/java/org/spongepowered/common/data/manipulator/mutable/block/SpongeWireAttachmentData.java

```diff
@@ -41,11 +42,18 @@
 
 import java.util.Map;
 
-public class SpongeWireAttachementData extends AbstractData<WireAttachmentData, ImmutableWireAttachmentData> implements WireAttachmentData {
+public class SpongeWireAttachmentData extends AbstractData<WireAttachmentData, ImmutableWireAttachmentData> implements WireAttachmentData {
 
     private Map<Direction, WireAttachmentType> wireAttachmentMap;
 
-    public SpongeWireAttachementData(Map<Direction, WireAttachmentType> attachmentMap) {
+    public SpongeWireAttachmentData() {
+        this(ImmutableMap.of(Direction.NORTH, WireAttachmentTypes.NONE,
```

Comment:
This map can be stored in ` final static` field and referenced here.

Reply:
Fixed.